eslint.config.ts

```diff
@@ -133,4 +148,41 @@ export default defineConfig(
 			"n/no-unsupported-features/node-builtins": "off",
 		},
 	},
+	{
+		extends: [markdownLinks.configs.recommended],
+		files: ["**/*.md"],
+		ignores: ["CHANGELOG.md"],
+		rules: {
+			"markdown-links/no-dead-urls": [
+				"error",
+				{
+					checkAnchor: false,
+					ignoreUrls: [
+						// npm gets rate-limited quickly: https://github.com/ota-meshi/eslint-plugin-markdown-links/issues/42
+						String.raw`/^https:\/\/(?:www\.)?npmjs.com\/.*/`,
+						// In case a not yet merged rule doc page is linked
+						String.raw`/^https://(?:www\.)?github.com/JoshuaKGoldberg/eslint-plugin-package-json/blob/[^/]+/docs/rules/[\w-]+.md([?#].*)?$/i`,
+					],
+					maxRetries: 3,
+					timeout: 5000,
+				},
+			],
+		},
+	},
+	{
+		files: ["package.json"],
+		languageOptions: {
+			parser: jsoncEslintParser,
+		},
+		plugins: {
+			// @ts-expect-error types mismatch
+			"node-dependencies": nodeDependencies,
+		},
+		rules: {
+			"node-dependencies/no-deprecated": [
+				"error",
+				{ devDependencies: true },
+			],
+		},
+	},
```

Comment:
Since this targets the same file as the config on line 99, can they be combined?